model/src/main/proto/javaclasses/mealorder/q/projections.proto

```diff
@@ -279,5 +275,5 @@ message UserSpendings {
 
     UserId id = 1;
 
-    int64 amount = 2;
+    spine.money.Money amount = 2;
 }
```

Comment:
Please tune IDEA to add empty new line on save.

Reply:
Enabled IDEA setting, thank you for this remark.